src/toil/wdl/wdltoil.py

```diff
@@ -1813,14 +1832,20 @@ def main() -> None:
     parser = argparse.ArgumentParser(description='Runs WDL files with toil.')
     addOptions(parser, jobstore_as_flag=True)
 
-    parser.add_argument("wdl_uri", type=str, help="WDL document URI")
-    parser.add_argument("inputs_uri", type=str, help="WDL input JSON URI")
+    parser.add_argument("wdl_uri", type=str,
+                        help="WDL document URI")
+    parser.add_argument("inputs_uri", type=str, nargs='?',
+                        help="WDL input JSON URI")
+    parser.add_argument("--input", "-i", dest="inputs_uri", type=str,
+                        help="WDL input JSON URI")
```

Comment:
`inputs_uri` already supports multiple arguments.  Do we need an `--input` option to supplement this as well?  I worry that this might be confusing to users.

Reply:
I think we need this to match the syntax expected from a WDL runner in the conformance tests, so we can avoid defining yet another syntax for calling the runner over there, different from both MiniWDL and Cromwell.

You aren't going to be able to use multiple of it though, and `nargs='?'` for `inputs_uri` I think just means you can have 0 or 1.